helm/identity-account-mapper/account-mapper/templates/role.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ph-ee-identity-account-mapper-role
```

Comment:
no need to create roles for each service, we are using one role for all servcies now